client/me/profile-gravatar/index.jsx

```diff
@@ -1,27 +1,28 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	debug = require( 'debug' )( 'calypso:me:sidebar-gravatar' );
+import React from 'react';
+import debugFactory from 'debug';
 
 /**
  * Internal dependencies
  */
-var Gravatar = require( 'components/gravatar' ),
-	eventRecorder = require( 'me/event-recorder' );
+import Gravatar from 'components/gravatar';
+import eventRecorder from 'me/event-recorder';
 
-module.exports = React.createClass( {
+const debug = debugFactory( 'calypso:me:sidebar-gravatar' );
 
+export default React.createClass( {
 	displayName: 'ProfileGravatar',
 
 	mixins: [ eventRecorder ],
 
-	componentDidMount: function() {
+	componentDidMount() {
 		debug( 'The ProfileGravatar component is mounted.' );
```

Comment:
I know it isn't part of the goal of this PR but I wonder if does it make sense?
Removing this debug line allows us remove the `componentDidMount` method, the `debug` instance and finally get rid of importing `debug` component.

Again ... it's up to you. :-D